.github/workflows/build.yml

```diff
@@ -68,3 +68,15 @@ jobs:
         run: |
           lean --recursive --export=mathlib.txt src/
           leanchecker mathlib.txt
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: generate docs
+        if: github.repository == 'leanprover-community/mathlib' && github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.lean_version == 'leanprover-community/lean:nightly'
+        run: |
+          chmod +x ./scripts/deploy_docs.sh
```

Comment:
Please just make the file executable in the git repository.

Reply:
Looks like I did this and forgot to remove this line.